bin/wkhtmltopdf

```diff
@@ -50,4 +51,24 @@ unless File.exist? binary
         "(missing binary: #{binary})."
 end
 
-exec *$*.unshift(binary)
+raise 'chromium-browser binary not found.' unless system('which chromium-browser')
+
+uuid = SecureRandom.uuid
+# ENV['PWD'] is path to kings-landing
+temp_path = "#{ENV['PWD']}/tmp"
+
+html_path = "#{temp_path}/html_#{uuid}.html"
```

Comment:
There is an issue with /tmp. Chromium couldn't load the files even tho it has the access. It generates the PDF with the root folder structure. I looked at this issue with Marijan and we came up with ENV['PWD'] as a workaround. If you have any idea why this is happening or how to fix it please let me know, because I would also like to use /tmp.

Reply:
Ah wow, didn't expect that. Got on a call with Igor to see it in actions. Turns out it's part of snap security confinements. Lets just put the HTML files into a subdirectory in the tmp folder, instead of the root.

Reply:
Done. Placed it in tmp/pdf folder.